samples/typescript_sample/README.md

```diff
@@ -0,0 +1,29 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Note
+
+* Please copy this directory outside the main `sendbird-uikit-react` project before you run `npm install`
+* To get type definitions, add this:
+
+```json
+  "include": [
+    "node_modules/@sendbird/uikit-react/index.d.ts"
+  ]
+```
+ `./tsconfig.json`
+
+
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
```

Comment:
What does \ mean?

Reply:
Probably new line. this part is generated, I did not create it